babel.cfg

```diff
@@ -1,4 +1,4 @@
-[python: themes/pybossa-default-theme/**.py]
-[jinja2: themes/pybossa-default-theme/**/templates/**.html]
-[jinja2: themes/pybossa-default-theme/templates/home/_pybossa_text.html]
```

Comment:
Sorry, I should not have commited that file. It's something that makes it work on my local. Will revert it.